src/components/message.vue

```diff
@@ -0,0 +1,517 @@
+                    <button type='button' :disabled='isEditable' @click='() => { setIsEdit(true) }' class='edit_comment_state_button edit_btn'><Edit /> Edit</button>
+                </div>  
+                <button v-else class='delete_edit_buttons_container reply_btn' @click='() => { setIsReplying(true) }' type='button'><img src='../photos/Reply.svg' alt='reply' width='14' height='13' /><span class='reply_text'>Reply</span> </button>
+                <button v-if='isEditable' class='udate_comment_text_btn' @click='() => {
+    setIsEdit(false)
+    changeCommentText({
+        id: comment.postId,
+        newCommentText: textareaValue
+    })
+    
+}'>UPDATE</button>
+{{console.log(allComments)}}
+                <div class='modal_container'>
+                    <div
+                        class='overlay'
+                        v-if='showModal'
+                    >
+                        <div class='modal'>
+                            <h3 class='modal_header'>Delete comment</h3>
+                            <p class='modal_text'>Are you sure you want to delete this comment? This will remove the comment and can’t be undone.</p>
+                            <div class='modal_buttons'>
+                                <button class='modal_btn modal_cancel' type='button' @click='() => { setShowModal(false) }'>NO, CANCEL</button>
+                                <button class='modal_btn modal_delete' type='button' @click='() => {
+    setShowModal(false)
+    deleteComment(comment.postId)
+}'>YES, DELETE</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            </div >
+            <div class='reply_container'>
+                    <form v-if='isReplying' @submit.prevent='addAnswerHandler' class='answer_form'>
+                        <textarea :value='replyText' @input='(newText) => { setReplyText(newText.target.value) }' class='message_textarea' name='message' id='message' cols='30' rows='10' placeholder='Add a reply…' required />
+                        <img class='avatar' width='32' height='32' :src='require(`../photos/photosForProfile/${currentUser.pathToPhoto}`)' alt='avatar' />
+                        <button class='send_message' type='submit'>REPLY</button>
+                    </form>
+                <div v-if='comment.answers && comment.answers.length !== 0' v-for='answer in comment.answers' :key="answer.postId">
+                    <message   :comment='answer' />  
+                </div>
+                
+            </div>
+
+
+        
+
+</template>
+<script>
+import moment from 'moment';
+import { v4 } from 'uuid';
+import IsToAddressTheUser from './isToAdressTheUser';
+import { mapState, mapMutations } from 'vuex';
+import Delete from '../icons/delete';
+import Edit from '../icons/edit';
+import message from './message';
+export default {
+    components: {
+        IsToAddressTheUser, Delete, Edit, message
```

Comment:
component names must start from capital letter
```suggestion
        IsToAddressTheUser, Delete, Edit, Message
```